### DISCUSSION TITLE
Argocd app wait hang forever

### DISCUSSION THREAD

**Discussion by uanid** [argoproj/argo-cd#6602] [General]
I want wait Application delete until finish

First, enter `argocd app delete my-app --cascade=true --propagation-policy=foreground`
Second, enter `argocd app wait my-app --suspended` or `--suspended --operation=false` or `--suspended --operation=false --health`

After Application deleted, but `argocd app wait` hang forever.

How do I know when `Application is deleted` or `delete operation finished`?

**Comment by wtam2018** [comment]
`suspended` is a health status.  Being in `suspended` status does not mean the app is deleted.   I am sure there is a command line to wait for an app to finish deleting.

**Reply by uanid** [reply]
Thank you for your comment
